Sources/_InstructionCounter/src/InstructionsExecuted.c

```diff
@@ -21,3 +30,8 @@ uint64_t getInstructionsExecuted() {
   }
   return 0;
 }
+#else
+uint64_t getInstructionsExecuted() {
```

Comment:
Would be nice to see if we can do something cross platform here, but fine for now I suppose 🤷

Reply:
Yeah, I agree. But let’s wait to see if a need comes up first since this is just a testing utility anyway.